### ISSUE TITLE
JSON Patch support

### ISSUE THREAD

**Issue by dmitrizagidulin** [solid/solid#82]
Placeholder issue about whether Solid will ever provide support for the [JSON Patch](https://tools.ietf.org/html/rfc6902) verb. (Moved from [solid-spec/#39](https://github.com/solid/solid-spec/issues/39))

**Comment by dmitrizagidulin** [comment]
See also https://github.com/solid/solid/issues/54

**Comment by nicola** [comment]
Also, I see Json Merge Patch (which is less expensive) http://tools.ietf.org/html/rfc7396

**Comment by sandhawke** [comment]
Yes, I'm a big fan of merge patches, which in my code I call "overlays".   I like how they're trivially simple.   My only real problem with them is they can't be stored in data that they can patch.   That's because they can't patch data that uses explicit nulls, and they use explicit nulls.  So to store them I your main data, you have to transform them first, I guess.  If you support data with cycles, you need some transforms anyway.

**Comment by nicola** [comment]
(off topic) @sandhawke have a look at http://nicola.io/datacontainer-playground/ - it is basically a server that returns json with the "ldp:contains" pattern
